### PR TITLE
Sprite For Vulps 101

### PR DESCRIPTION
**Pull request by Kr8art**
## About the PR
Added a custom sprite for Lyndo's "Vulpkanin 101" book. 

## Why / Balance
Questionable life choices. 

## Technical details
- Added a book.rsi to DeltaV textures
- Created a new icon for the book
- Changed the books.yml of "Vulpkanin 101" to adopt the new icon

## Media
image

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
n/a

**Changelog**
:cl:
- tweak: Vulpkanin 101 now has a custom icon!

### CODE REVIEW

**Comment by Lyndomen** [comment]
hell yeah

File: Resources/Prototypes/DeltaV/Entities/Objects/Misc/books.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@
       color: "#961cc8"
     - state: detail_bookmark
       color: "#eee039"
-    - state: icon_question
+    - state: icon_kiss
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Objects/Misc/books.yml:41]: add the `sprite: DeltaV/...` to this layer so you dont have to duplicate the other states

File: Resources/Prototypes/DeltaV/Entities/Objects/Misc/books.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@
   description: You seriously didn't purchase a "For Dummies" book in 2425?
   components:
   - type: Sprite
-    sprite: Objects/Misc/books.rsi
+    sprite: DeltaV/Objects/Misc/books.rsi
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Objects/Misc/books.yml]: undo

File: Resources/Textures/DeltaV/Objects/Misc/books.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Base sprite taken at: https://github.com/tgstation/tgstation/commit/37fb6bc6dd20005775dde8d886f48f7722606b77 , splitted on layers by TheShuEd (github), icon_kiss by Kr8",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "cover_strong"
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Objects/Misc/books.rsi/meta.json]: remove other states and update copyright

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by Kr8art** [comment]
Addressed requested changes!

**Review by deltanedas — APPROVED** [review]

**Comment by deltanedas** [comment]
youre a secmain vulp arent you, you like arresting tiders

**Pull request merged**